src/tutorials/0008-introduction-to-libp2p/04.md

```diff
@@ -0,0 +1,21 @@
+---
+    title: "libp2p in the OSI Model"
+    type: "multiple-choice"
+---
+
+So where does libp2p fit into the current web?
+Usually, we look at the **OSI Model (Open Systems Interconnection Model)** to understand which layer a protocol tries to tackle.
+
+<div class="flex justify-center mv4">
+    <img src="/tutorial-assets/T0008L04-osi-model.png">
+</div>
+
+But the truth is the `OSI Model` as well as the `TCP/IP Model` are just conceptual models. The actual implementations in the current web do not follow these models exactly, but instead it's a mix, i.e., the `TCP/IP Protocol Suite`.
+
+In the end, these models have some shortcomings:
+
+- Actions are repeated through multiple layers (repeated work) because data can go through multiplexing multiple times, or authentication multiple times or even discovery multiple times - it's very inefficient.
+- Information is hidden between layers missing significant improvement opportunities and some protocols have been created with these opportunities in mind by trying to put together as many layers as possible into a single protocol module (an example of this is the [QUIC](https://www.chromium.org/quic) protocol, a multiplexed stream transport over `UDP`).
```

Comment:
What's a stream transport?

Reply:
I don't believe it's worth explaining it here. We just wanted to mention the fact that QUIC is good example of a protocol that was designed to stuff as much complexity into it to be able to perform optimisations.

Reply:
> I don't know how you usually take into account this less familiar concepts. I feel that it is good to throw them out in this context for completeness. But, we should probably find a good way of saying to the user: "It is fine if you are not familiar with this concepts, as we will explore them over the course"

The only problem with that is that we won't explain it, maybe just a quick simple definition. Basically the read should't be expecting to understand what this is now or ahead in the tutorial.
I think we can remove this detail after the comma.

Reply:
done here: d59e724e3f51b05ffa910cbc4e8e6c782ac89ab4